examples/data_download_multiple_assets.py

```diff
@@ -33,7 +33,8 @@
 wildfire_asset_type = 'basic_analytic_4b'
 
 
-async def download_and_validate(item_id, item_type_id, asset_type_id, client):
+async def download_and_validate(client, item_id, item_type_id, asset_type_id):
```

Comment:
I like the positioning of `client` here, like `self`. I think you mentioned that in a comment somewhere. Great call.